packages/wasi/lib/polyfills/bigint.ts

```diff
@@ -1,20 +1,8 @@
 // A very simple workaround for Big int. Works in conjunction with our custom
 // Dataview workaround at ./dataview.ts
 
-// Add Big int depending on the environment
-let exportedBigInt = Number;
-let globalThis: any = {
-  BigInt: {}
-};
-/*ROLLUP_REPLACE_NODE
-globalThis = global;
-ROLLUP_REPLACE_NODE*/
-/*ROLLUP_REPLACE_BROWSER
-globalThis = self;
-ROLLUP_REPLACE_BROWSER*/
-if (globalThis.BigInt) {
-  exportedBigInt = globalThis.BigInt;
-}
+var commonjsGlobal = typeof globalThis !== "undefined" ? globalThis : {};
```

Comment:
Can we the license here in a comment? https://github.com/rollup/rollup-plugin-commonjs

Also, we can change the `var` into a `const`

Reply:
No need to license. It's just a variable assignment (and it's not the same content, either).
Just improved the var to be const and improved var name